RNTester/e2e/test-init.js

```diff
@@ -8,7 +8,7 @@
 /* eslint-env jasmine */
```

Comment:
I moved this file because jest requires all files in the `__tests__` directory to have tests. I believe this is getting fixed soon but this makes things pass until then.